core/templates/pages/donate-page/donate-page.component.html

```diff
@@ -252,7 +252,7 @@ <h1 class="text-light oppia-donate-page-heading">
                 <picture>
                   <source type="image/webp" [srcset]="getImageSet('/splash/Dheeraj', 'webp')">
                   <source type="image/png" [srcset]="getImageSet('/splash/Dheeraj', 'png')">
-                  <img [src]="getStaticImageUrl('/splash/Dheeraj', 'png')"
+                  <img [src]="getImageSet('/splash/Dheeraj', 'png')"
```

Comment:
What is an "image set"? This is new to me and AFAIK isn't a standard term.

Reply:
Not my introduction. Maybe @kevintab95 can explain the naming here.

Reply:
Sorry, I missed this, Srijan is right, I introduced it in the donate page recently. We followed this convention in the splash page. It basically returns a string containing image URLs for different image sizes. See 

https://github.com/oppia/oppia/blob/80ff394bf278366f3e482da1eff86a53772fca3c/core/templates/pages/splash-page/splash-page.component.ts#L76-L82